docs/_docs/persistence/snapshots.adoc

```diff
@@ -131,15 +134,18 @@ include::{javaCodeDir}/Snapshots.java[tags=create, indent=0]
 == Restoring From Snapshot
 
 Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes. The detailed procedure
-looks as follows:
+replace persistence data and other files with the data from the snapshot, and restart the nodes. Prior to performing
```

Comment:
It feels like we are saying too much in the opening paragraph. Basically, we're listing some or most of the restore steps that are covered down below. 

How about shortening the opening paragraph to a single sentence and listing all the steps in the numbered-list below?

_"Currently, the data restore procedure has to be performed manually. Follow these step to restore a cluster from a snapshot:
1) Stop the cluster you intend to restore
2)..
N)
"_

Reply:
I've extended the list as you suggested.